js/ai.js

```diff
@@ -67,6 +67,14 @@ export default {
         this.test = 0;
         this.foundSet = false;
 
+        // have ai check each time it resumes if game should end; this logic used to be in board.js before card draw
+        let numValidSets = this.findValidSets().length
+        if (isDev)  console.log(`ai sets found: ${numValidSets}, cards left: ${deck.stock.length`);
```

Comment:
This is a syntax error - missing the closing `}` at the end of the line. i.e. should be ``.length}`);``

Reply:
Good catch @zachkirsch! Thanks. Just fixed on master. :)

Reply:
Thanks for catching that!  I had it correct on my local copy, not sure how I messed that up pasting over to github.  Much appreciated, I'll fix my own branch too